components/ArticleRecirculation.vue

```diff
@@ -90,7 +90,7 @@ onBeforeUnmount(() => {
           <v-card
             class="hidden xl:flex article-md mod-vertical mod-large mb-5"
             :image="useImageUrl(articleMd.listingImage)"
-            :title="articleMd.title"
+            :title="articleMd.listingTitle || articleMd.title"
```

Comment:
listingTitle already uses the regular title as a fallback so you should only need to use `articleMd.listingTitle` here

Reply:
Ah good to know! I'll make that change in my next PR